examples/hello_ocean.jl

```diff
@@ -0,0 +1,43 @@
+# # Hello, ocean!
+#
+# > How inappropriate to call this planet Earth, when it is quite clearly _Ocean_.
+#
+#   --Arthur C. Clark
+
+using Oceananigans, Oceananigans.Grids, Plots
+
+grid = RegularCartesianGrid(size = (64, 1, 64),
+                            x = (-5, 5), y = (-5, 5), z = (-3, 1),
+                            topology = (Periodic, Periodic, Bounded))
+
+model = IncompressibleModel(grid = grid,
+                            architecture = CPU(),
+                            advection = Oceananigans.Advection.WENO5(),
+                            timestepper = :RungeKutta3,
+                            tracers = :b,
+                            buoyancy = BuoyancyTracer(),
+                            closure = IsotropicDiffusivity(ν=1e-3, κ=1e-3))
+
+@info "Simulating the ocean with" model
+
+Σ(ξ) = (1 - tanh(ξ)) / 2
+Π(ξ, δ=1) = (Σ(ξ - δ) - Σ(ξ + δ)) / 2
+
+set!(model,
+     u = (x, y, z) -> Σ(-z),
+     b = (x, y, z) -> - Π(4x, 1) * Σ(32z))
```

Comment:
I might give these more self-explanatory names as I imagine they look quite cryptic, especially to new users.